src/search/result.ts

```diff
@@ -109,8 +106,8 @@ export class IndexedDoc<T extends TigrisCollectionType> {
 		return this._meta;
 	}
 
-	static from<T>(resp: ProtoSearchHit | ProtoIndexDoc, config: TigrisClientConfig): IndexedDoc<T> {
```

Comment:
Since both `IndexDoc` and `SearchHit` were structurally identical, the deserialization was same for both.